### ISSUE TITLE
Could curl_multi_init be used to parallel request

### ISSUE THREAD

**Issue by narchibald** [vgrem/phpSPO#252]
Could curl_multi_init be used by the clientrequest to execute multiple requests at a time?
https://www.php.net/manual/en/function.curl-multi-init.php
I need to load the author for multiple SharePoint files and I see that the load request for each file is in sequence? I have trying requesting the author when populating the folder items but it looks as though that data is not returned even though the request looks correct. So since I need to populate the data for each file in the folder it would be nice to have these requests done in parallel.
